rust/relay/src/main.rs

```diff
@@ -60,6 +60,11 @@ struct Args {
     /// If omitted, we won't connect to the portal on startup.
     #[arg(env = "FIREZONE_TOKEN")]
     token: Option<SecretString>,
+    /// Token generated by the portal to authorize websocket connection.
+    ///
+    /// If omitted, we won't connect to the portal on startup.
+    #[arg(env = "FIREZONE_NAME")]
```

Comment:
Does this require a matching terraform update to name the relays for staging / prod?

Reply:
It is entirely optional for now so any update to terraform can be made after this PR as well.

Reply:
Is it a name or a token? (Happened to see this in the PR list and wondered if it was more like the firezone-id or more like the account slug)

Reply:
cc @AndrewDryga